docs/angular/your-first-app.md

```diff
@@ -109,11 +110,18 @@ npm install @ionic/pwa-elements
 
 Next, import `@ionic/pwa-elements` by editing `src/main.ts`.
 
-```tsx
+```ts
+import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
+import { AppModule } from './app/app.module';
+// CHANGE: Add the following import.
 import { defineCustomElements } from '@ionic/pwa-elements/loader';
 
-// Call the element loader before the bootstrapModule/bootstrapApplication call
+// CHANGE: Call the element loader before the `bootstrapModule` call.
```

Comment:
Instead of doing `// CHANGE` should we be using a `diff` code block here? Or would that be more confusing later on to copy/paste?